oss_src/unity/python/sframe/cython/cy_flexible_type.pyx

```diff
@@ -1223,8 +1230,9 @@ cdef inline tr_datetime64_to_ft(flexible_type& ret, v):
     # Since flexible type datetime only goes down to microseconds, convert to
     # this. If higher resolution, this will truncate values
     cdef object as_py_datetime = v.astype('M8[us]').astype('O')
-    as_py_datetime = as_py_datetime.replace(tzinfo=GMT(0))
-    tr_datetime_to_ft(ret, as_py_datetime)
+    if as_py_datetime is not None:
+        as_py_datetime = as_py_datetime.replace(tzinfo=GMT(0))
+        tr_datetime_to_ft(ret, as_py_datetime)
```

Comment:
I wasn't able to assign to ret from within that function. I couldn't figure out why. This is why I assigned ret to FLEX_UNDEFINED if a NaT was detected at line 1434, but I could just assign it regardless so that ret is always FLEX_UNDEFINED before this function is called. Then ret just won't be changed if as_py_datetime is None. Does that sound good?